IndustrialUnit.WpfUI/ViewModels/EquipmentViewModel.cs

```diff
@@ -0,0 +1,165 @@
+﻿using IndustrialUnit.WpfUI.Models;
+using IndustrialUnitDatabase;
+using System;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Windows.Input;
+
+namespace IndustrialUnit.WpfUI.ViewModels
+{
+  public class EquipmentViewModel : BaseViewModel
+  {
+    private ObservableCollection<EquipmentViewProperties> _equipments;
+    public ObservableCollection<EquipmentViewProperties> Equipments
+    {
+      get
+      {
+        return _equipments;
+      }
+      set
+      {
+        _equipments = value;
+        OnPropertyChanged();
+        SelectedEquipment = new EquipmentViewProperties();
+      }
+    }
+
+    private EquipmentViewProperties _selectedEquipment;
+    public EquipmentViewProperties SelectedEquipment
+    {
+      get
+      {
+        return _selectedEquipment;
+      }
+      set
+      {
+        _selectedEquipment = value;
+        OnPropertyChanged();
+      }
+    }
+
+    private string _messageToView;
+    public string MessageToView
+    {
+      get
+      {
+        return _messageToView;
+      }
+      set
+      {
+        _messageToView = value;
+        OnPropertyChanged();
+      }
+    }
+
+
+    public static ObservableCollection<EquipmentViewProperties> GetEquipments()
+    {
+      ObservableCollection<EquipmentViewProperties> equipmentCollection = new();
+      var getEquipment = SQLiteDataAccess.GetAll("Equipment");
```

Comment:
Már írtam ott is, de Dapper. A data access layernek már típusos adatot érdemes visszaadnia, hogy a viewmodel-ben már ne kelljen táblanevekkel és egyebekkel foglalkozni.